tests/virt/utils.py

```diff
@@ -502,3 +503,19 @@ def get_non_terminated_pods(client, node):
             field_selector=f"spec.nodeName={node.name},status.phase!=Succeeded,status.phase!=Failed",
         )
     )
+
+
+def get_boot_time_for_multiple_vms(vm_list):
+    boot_time_dict = {}
+    for vm in vm_list:
+        boot_time_dict[vm.name] = get_vm_boot_time(vm=vm)
+    return boot_time_dict
```

Comment:
You can use dict comprehension here, also needs to add typing
```suggestion
def get_boot_time_for_multiple_vms(vm_list):
    return {vm.name: get_vm_boot_time(vm=vm) for vm in vm_list}
```